### ISSUE TITLE
slider not shown in @interact

### ISSUE THREAD

**Issue by azerad** [3-manifolds/Sage_macOS#42]
in a jupyter notebook with sagemath9.6 just installed  on a mac book macOS 12.1
 no slider shown and no error message in the following piece of code
#####
@interact
def exemple(x=2):
    return 1 +x
########

**Comment by NathanDunfield** [comment]
I can reproduce this with the Chrome browser on macOS 11.6; in contrast, `@interact` works with SageMath 9.5 on the same system.  Possibly this is a regression related to [this discussion](https://groups.google.com/g/sage-support/c/FlLA_7tUb0M).

**Comment by NathanDunfield** [comment]
Yes, it is a regression.  The following terminal commands fixed it on my system:
```
cd /var/tmp/sage-9.6-current/venv/share/jupyter/nbextensions/
mkdir -p  widgets/notebook
cd widgets/notebook/
ln -s ../../jupyter-js-widgets js
```
Marc (@culler) can you confirm that this is the correct approach?

**Comment by azerad** [comment]
Thanks  for the suggested fix which works for me (macOS 12.1 sagemath 9.6)

:-)

P


Le 13/06/2022 à 15:28, Nathan Dunfield a écrit :

**Comment by NathanDunfield** [comment]
Reopened to remind us to fix this in the release.

**Comment by NathanDunfield** [comment]
No worries, many thanks for reporting this!

**Comment by culler** [comment]
I believe that this is now fixed in v1.4.1.  That is a prerelase at the moment but will be changed to a release in a few days if everything continues to look OK.  In any case, I think we can close this issue now.

**Comment by NathanDunfield** [comment]
Interact works for me in v1.4.1 without any issues.
